src/test/java/hw2/ex2/Ex2Test.java

```diff
@@ -0,0 +1,99 @@
+package hw2.ex2;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class Ex2Test {
+    private WebDriver driver;
+    String url = "https://jdi-testing.github.io/jdi-light/index.html";
+    String url2 = "https://jdi-testing.github.io/jdi-light/different-elements.html";
+    String login = "Roman";
+    String pass = "Jdi1234";
+    String user = "ROMAN IOVLEV";
+    String[][] elementsTest = {{"Water", "cbox"}, {"Wind", "cbox"}, {"Selen", "radio"}, {"Yellow", "dd"}};
+
+    @BeforeTest
+    public void setUp() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(1, TimeUnit.SECONDS);
+        driver.manage().timeouts().pageLoadTimeout(3, TimeUnit.SECONDS);
+    }
+
+    @Test
+    public void ex1Test() {
+        SoftAssert softAssert = new SoftAssert();
+        //1. open site
+        driver.get(url);
+        softAssert.assertEquals(driver.getCurrentUrl(), url);
+
+        //2. Browser title equals "Home Page"
+        softAssert.assertEquals(driver.getTitle(), "Home Page");
+
+        //3. Perform login
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.id("name")).sendKeys(login);
+        driver.findElement(By.id("password")).sendKeys(pass);
+        driver.findElement(By.id("login-button")).click();
```

Comment:
looks the same as in Ex1Test. If so, then common parts can be moved to the parent class

Reply:
Parent abstract class LoginTest was added